src/VirtualClient/VirtualClient.Main/VirtualClient.Main.csproj

```diff
@@ -27,6 +27,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <Folder Include="Terminal\" />
```

Comment:
maybe lower case? and also not seeing any file under Terminal

Reply:
I meant to delete this folder. Thanks for the pointer.